L/libjulia/common.jl

```diff
@@ -77,10 +86,11 @@ function build_julia(ARGS, version)
     override OS=Linux
     EOM
 
+    LLVM_CXXFLAGS="-I${prefix}/include -fno-exceptions -fno-rtti -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS"
```

Comment:
eventually we should get these from `llvm-config` from a HostDependency now that we have those

Reply:
Agreed! But i have no idea (yet) how to use those, and so I hope this can wait (or someone else does it)

Reply:
I've added a TODO comment stating this